tests/check_framework/test_translation.py

```diff
@@ -0,0 +1,59 @@
+import re
+
+from django.conf import settings
+from django.core.checks.translation import (
+    LANGUAGE_CODE_PATTERN, check_setting_language_code,
+)
+from django.test import SimpleTestCase
+
+
+class TranslationCheckTests(SimpleTestCase):
+
+    def test_valid_language_code_format_ll_only(self):
+        settings.LANGUAGE_CODE = "en"
```

Comment:
We should use `override_settings` for this, to ensure we clean up after ourselves once the test is complete, avoiding breaking unrelated tests.

Reply:
done